src/index.js

```diff
@@ -263,6 +263,8 @@ function main() {
       const anchor = getMarkdownAnchor(err.message);
       openUrl(`${pkg.homepage}/blob/master/docs/errors.md${anchor}`);
     }
+  } finally {
+    exit(); // eslint-disable-line
```

Comment:
Just to pint out that this PR will also declaratively exit the program after it's done.
Just a formality actually, and for clarity, but doesn't affect anything really.